src/util/EditionCardUtils.tsx

```diff
@@ -183,6 +183,7 @@ export default class EditionCardUtils {
   static getReadOnlineLink = (item: ApiItem) => {
     const getReadLink = (item: ApiItem, mediaType: string) => {
       if (!item || !item.links) return undefined;
+      /// TODO: feature `flags` object indicator
```

Comment:
nit: Perhaps call this "use `item.link.flags`" so it's not easily confused with the "Feature Flags" feature, which has been on our to-do list for a while.